lib/pager.gi

```diff
@@ -136,6 +129,31 @@ BindGlobal("PAGER_BUILTIN", function( lines )
     return;
   fi;
 
+  if IsRecord(r) then
+    if IsBound(r.formatted) then
+      formatted := r.formatted;
+    fi;
+    if IsBound(r.start) then
+      if IsInt(r.start) then
+        linepos := r.start;
+      elif IsString(r.start) and StartsWith(r.start, "/Obj *Func") then
+        # regular expression most likely specified by `PageSource`
+        nam := r.start{[7..Length(r.start)]};
+        linepos:= PositionsProperty(lines,
+                    l -> PositionSublist(l, nam) <> fail);
+        linepos:= First(linepos,
+                    i -> PositionSublist(lines[i], "Obj")
+                         < PositionSublist(lines[i], nam) and
+                         ForAll(lines[i]{[PositionSublist(lines[i], "Obj")+3
+                                   .. PositionSublist(lines[i], nam)-1]},
+                           x -> x = ' '));
+      fi;
```

Comment:
Should we issue an error when `r.start` is unsupported?
```suggestion
      else
        Error("unsupported r.start value: ", r.start);
      fi;
```

But I am also OK with ignoring it